iron-overlay-behavior.html

```diff
@@ -155,11 +155,8 @@
       return Polymer.dom(this).querySelector('[autofocus]') || this;
     },
 
-    registered: function() {
-      this._backdrop = document.createElement('iron-overlay-backdrop');
```

Comment:
oh, and we can also remove it from here (the line 14), since the import of `iron-overlay-manager` will import the backdrop :)